frontend/src/App.css

```diff
@@ -187,6 +186,71 @@ body {
 }
 
 .footer button:active {
-    filter: brightness(150%);
-    box-shadow: 0 0 5px var(--persian-pink);
-}
\ No newline at end of file
+  filter: brightness(150%);
+  box-shadow: 0 0 5px var(--persian-pink);
+}
+/*Card*/
+.cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-gap: 20px;
+  margin: 1em auto;
+}
+
+.card {
+  display: grid;
+  padding: 5px;
+  border: 2px solid #999;
+  border-radius: 3px;
+  background-color: #fff;
+  width: 300px;
+}
+
+.card img {
+  object-fit: cover;
+  width: 100%;
+}
+
+.card h2 {
+  margin: 0;
+  padding: 0.5rem;
+}
+
+.card .content {
+  padding: 0.5rem;
+}
+
+.card footer {
+  background-color: rgb(3, 60, 9);
```

Comment:
This color isn't from the color palette. Please use CSS variable `--hunter-green`.